connexion/decorators/security.py

```diff
@@ -228,12 +230,29 @@ def wrapper(request, required_scopes):
     return wrapper
 
 
+def get_cookie_value(cookies, name):
+    '''
+    Returns cookie value by its name. None if no such value.
+    :param cookies: str: cookies raw data
+    :param name: str: cookies key
+    '''
+    cookie_parser = SimpleCookie()
+    cookie_parser.load(cookies)
+    if name in cookie_parser.keys():
```

Comment:
is it possible to do this in a pythonic way?
```
try:
    return cookie_parser[name].value
except KeyError:
    return None
```

Reply:
Hi! Thank you!
I made the change, as you had requested.

```suggestion
def get_cookie_value(cookies, name):
    '''
    Returns cookie value by its name. None if no such value.
    :param cookies: str: cookies raw data
    :param name: str: cookies key
    '''
    cookie_parser = http_cookies.SimpleCookie()
    cookie_parser.load(str(cookies))
    try:
        return cookie_parser[name].value
    except KeyError:
        return None

```
I added str(cookies) to your example to convert the "cookies" variable to string to make comparison in python Cookies.py possible:
```
def load(self, rawdata):
        """Load cookies from a string (presumably HTTP_COOKIE) or
        from a dictionary.  Loading cookies from a dictionary 'd'
        is equivalent to calling:
            map(Cookie.__setitem__, d.keys(), d.values())
        """
        if type(rawdata) == type(""):
            self.__ParseString(rawdata)
        else:
            # self.update() wouldn't call our custom __setitem__
            for k, v in rawdata.items():
                self[k] = v
        return
    # end load()
```

P.S. I am not good at git. I was not able to make correct addtion to your request with some kind of link to my changes. I am really sorry.
https://github.com/ErmakovDmitriy/connexion/commit/7e312eedacf80a7a3b267b079c8236a8e0b80b06